app/src/androidTest/java/com/github/studydistractor/sdp/ui/DistractionScreenTest.kt

```diff
@@ -43,4 +43,35 @@ class DistractionScreenTest {
         composeRule.onNodeWithTag("description").assertExists()
         composeRule.onNodeWithTag("description").assert(hasText(description))
     }
+
+    @Test
+    fun buttonToCompleteActivityExistsAndHasCorrectText() {

```

Comment:
Just a small remark: remove the To (`buttonToCompleteActivityExistsAndHasCorrectText` becomes `buttonCompleteActivityExistsAndHasCorrectText`)